apps/web/src/app/(signing)/sign/[token]/complete/page.tsx

```diff
@@ -62,96 +68,112 @@ export default async function CompletedSigningPage({
 
   const sessionData = await getServerSession();
   const isLoggedIn = !!sessionData?.user;
+  const callToActionCheck = !isLoggedIn && NEXT_PUBLIC_DISABLE_SIGNUP?.toString() !== 'true';
```

Comment:
`toString()` because people might use a boolean value in the `.env` file...

Reply:
I believe everything in an env file gets passed through as a string